### PR TITLE
Add Next.js SDK's performance monitoring docs

### PR DESCRIPTION
**Pull request by iker-barriocanal**
Docs for the new performance monitoring of the Next.js SDK.

### CODE REVIEW

**Comment by iker-barriocanal** [comment]
@rahulchhabria

**Review by AbhiPrasad — COMMENTED** [review]
What do you think about switching from `frontend` -> `client-side` and `backend` -> `server-side`? I think that better matches to how nextjs refers to those concepts, but I'm not sure it is as clear as just saying frontend, backend.

**Comment by iker-barriocanal** [comment]
> What do you think about switching from `frontend` -> `client-side` and `backend` -> `server-side`? I think that better matches to how nextjs refers to those concepts, but I'm not sure it is as clear as just saying frontend, backend.

Yes @AbhiPrasad, in fact, that's the language we've been using in the SDK. Updated.

File: src/includes/getting-started-primer/javascript.nextjs.mdx [hunk 1]
```diff
@@ -12,6 +12,7 @@ Features:
 - Events [enriched](/platforms/javascript/enriching-events/context/) with device data
 - [Breadcrumbs](/platforms/javascript/enriching-events/breadcrumbs/) created for outgoing HTTP request with XHR and Fetch, and console logs
 - [Release Health](/product/releases/health/) for tracking crash-free users and sessions
+- Automatic [Performance Monitoring](/product/performance/) for both the client and server
```
Comment (AbhiPrasad) [inline src/includes/getting-started-primer/javascript.nextjs.mdx:15]: Can we put something like `**new in 6.5.0**`?

**Review by AbhiPrasad — CHANGES_REQUESTED** [review]
We need some indication that performance monitoring is only available for the SDK v6.5.0, otherwise people will def get confused. I left a comment on for one place where we can add it, but there might be more places.

**Review by HazAT — CHANGES_REQUESTED** [review]
We should also add it to the wizard `tracesSampleRate`

**Review by HazAT — APPROVED** [review]
Sorry, it's already there 😓

**Pull request merged**